### DISCUSSION TITLE
Allow certain version only if other package matches its own pattern

### DISCUSSION THREAD

**Discussion by martokarski** [renovatebot/renovate#22380] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi, I have a very specific case. Consider package A that is versioned X.X.X. Then I have package B, that is versioned X.X.X_[1-9]. Those both packages are related, but package A is always released first and after couple of weeks package B is released with the same version. For example we had release of package A 5.3.27 and soon we will have package B 5.3.27_1.

In my Maven project I defined property for package A, so Renovate bot can easily update it automatically. I'm using that property also to define version of package B (${packageA.version}_1). However Renovate doesn't care about my package B and updates version as soon as package A is released. It causes build failures, because package B in that version isn't yet released.

Can I somehow configure Renovate to wait with the update of package A until specified version of package B will be released? Like in the example above I would like that Renovate would update my project only when package B version 5.3.27_1 will be released.

### What you tried so far.

I tried to go through the documentation of `packageRules` bud didn't find anything useful.

**Comment by rarkins** [comment, marked as answer]
Sounds like it could be solved this way: https://github.com/renovatebot/renovate/issues/19645#issuecomment-1369783891 (non-implemented)

i.e. you set minimumUpdates=2 and it would mean the branch isn't created unless both are present

**Reply by martokarski** [reply]
Wow, thank you for a quick reply :)

It is sad Renovate doesn't have such option, but I understand it might be difficult to implement :(

**Reply by rarkins** [reply]
PRs welcome
